web/templates/docs/publish.html.eex

```diff
@@ -93,6 +93,8 @@ You are required to confirm your email to access your account, a confirmation em
     [app: :postgrex,
      version: "0.1.0",
      elixir: "0.13.0",
```

Comment:
Yeah, we should probably do "~> 1.0" here. But I am generally ok with keeping it relatively old. :)